subsys/net/lib/nrf_cloud/src/nrf_cloud_cell_pos.c

```diff
@@ -38,7 +38,8 @@ int nrf_cloud_cell_pos_request(const struct lte_lc_cells_info * const cells_inf,
 
 	if (cells_inf) {
 		/* Add provided cell info to the data obj */
-		err = nrf_cloud_format_cell_pos_req_json(cells_inf, 1, data_obj);
+		err = nrf_cloud_format_cell_pos_req_json(cells_inf, cells_inf->ncells_count + 1,
```

Comment:
the hard coded 1 is for the number of cell info items, not neighbor cells.  Not sure why this fixes the issue.  Probably something wrong in the codec.

Reply:
No, something is wrong with how much of the codec I read through.  I forgot that the cell info structure contains the information about the connected cell, and that the array of neighbors is further down in the structure.  So I didn't look down far enough to see the for loop on the neighbors.

Thanks.